dataprofiler/profilers/profile_builder.py

```diff
@@ -1519,7 +1520,10 @@ def __init__(
         # Structured specific properties
         self.row_has_null_count = 0
         self.row_is_null_count = 0
-        self.hashed_row_dict: dict = dict()
+        if options.hll_row_hashing:
+            self.hashed_row_dict: HyperLogLog = HyperLogLog(10, sparse=False)
```

Comment:
Arbitrary register count (2 ^ 10). Curious how we want to make this scale with data input. I'm thinking `2 * log2(number of records)` might be good 🤷

Reply:
we can allow users to set the values via the `ProfileOptions` being made for this but for choice of value, is there a way to choose relative to an accuracy somehow? as oppose to the value itself? i'm not sure we will always know the true data size. What's the space req for the largest value? is it problematic

Reply:
we might want to rename this variable, open to thoughts.

Reply:
Can we control the seed so it isn't random every time? set via options, default to something else.